tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -18,6 +18,9 @@
 #define _WCHAR_H_CPLUSPLUS_98_CONFORMANCE_
 #endif
 
+// clang-format off
+// Windows include ordering is fragile.
```

Comment:
The other places the comment is "Includes on Windows are highly order dependent."

(I have a nit here in that it's not because the code is being compiled on Windows that makes it fragile, it's because the Windows SDK header files are order dependent.)

Reply:
It's more than just the Windows SDK headers. It is also the Windows-specific DXC code that is order dependent and extremely fragile. DXILConv in particular is a crazy mess.